image/templates/helm/stackrox-central/templates/01-central-08-db-configmap.yaml

```diff
@@ -0,0 +1,939 @@
+{{- include "srox.init" . -}}
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: central-db-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "srox.labels" (list . "configmap" "central-db-config") | nindent 4 }}
+  annotations:
+    {{- include "srox.annotations" (list . "configmap" "central-db-config") | nindent 4 }}
+data:
+  postgresql.conf: |
+    {{- if ._rox.central.enableCentralDB }}
```

Comment:
I'm pretty sure this is misplaced, because you only `{{- end }}` it after `pg_hba.conf`, and I don't see why one file would be conditional, whereas the other one would exist unconditionally (but possibly be empty).

I think what you want here is to make the creation of the entire `ConfigMap` resource conditional, no?

Reply:
correct